lib/matplotlib/font_manager.py

```diff
@@ -838,7 +851,8 @@ def set_weight(self, weight):
         Set the font weight.  May be either a numeric value in the
         range 0-1000 or one of 'ultralight', 'light', 'normal',
         'regular', 'book', 'medium', 'roman', 'semibold', 'demibold',
-        'demi', 'bold', 'heavy', 'extra bold', 'black'
+        'demi', 'bold', 'heavy', 'extra bold', 'black'.
+        Default: :rc:`font.weight`
```

Comment:
Should probably change this to numpydoc.

Reply:
Yes, I was a bit lazy as I couldn't really figure out what the right way to transform "numeric value in the range 0-1000" in a sensible way to that format. float and then mention the range below?

Reply:
That's not a type; just put it in the text description.